subsys/bluetooth/controller/ll_sw/nordic/lll/lll_prof.c

```diff
@@ -111,8 +113,50 @@ void lll_prof_cputime_capture(void)
 }
 
 void lll_prof_send(void)
+{
+	struct node_rx_pdu *rx;
+
+	/* Generate only if spare node rx is available */
+	rx = ull_pdu_rx_alloc_peek(3);
```

Comment:
Why do you check if there are available 3 node_rx instances? Where the number comes from?

Reply:
This needs to get defined in a define, but in a separate PR.

Otherwise, it is to check if there are 3 free rx node buffers wherein 1 will be used for generating the current profiling notification, 2 others will remain back in free pool for use in any active connection (1 to generate received ACL data, and 1 reserved to maintain ACL connection).